vue-app/src/api/recipient-registry-simple.ts

```diff
@@ -39,17 +39,19 @@ export async function getProjects(
       // Invalid metadata
       continue
     }
-    if (endBlock && event.blockNumber >= endBlock) {
+    const addedAt = (event.args as any)._timestamp.toNumber()
+    if (endTime && addedAt >= endTime) {
       // Hide recipient if it is added after the end of round
       project.isHidden = true
     }
     const removed = recipientRemovedEvents.find((event) => {
       return (event.args as any)._recipientId === project.id
     })
     if (removed) {
-      if (!startBlock || startBlock && removed.blockNumber <= startBlock) {
-        // Start block not specified
-        // or recipient had been removed before start block
+      const removedAt = (removed.args as any)._timestamp.toNumber()
+      if (!startTime || startTime && removedAt <= startTime) {
```

Comment:
Same comment here as above. It also makes me wonder if this can be made more DRY since we're seeing this code again.

```suggestion
      if (!startTime || removedAt <= startTime) {
```

Reply:
>It also makes me wonder if this can be made more DRY since we're seeing this code again.

Not worth the increased complexity at the moment.